### PR TITLE
chore: bump wasmedge to 0.12.1 and wasmedge-sdk to 0.8.1

### PR DESCRIPTION
**Pull request by CaptainVincent**
1. bump new version
2. remove async feature relative to #72
    > According to the information I received from my colleague, there is no urgent need for a fork at the moment, so let me disable it here first.

### CODE REVIEW

File: crates/containerd-shim-wasmedge/src/executor.rs [hunk 1]
```diff
@@ -42,17 +42,17 @@ impl Executor for WasmEdgeExecutor {
             .build()?;
 
         // create a vm with the config settings
-        let mut vm = Vm::new(Some(config))?;
+        let mut vm = VmBuilder::new().with_config(config).build()?;
 
         // initialize the wasi module with the parsed parameters
-        let mut wasi_instance = vm.wasi_module()?;
-        wasi_instance.initialize(
+        let wasi_module = vm.wasi_module_mut().ok_or("Not found wasi module").unwrap();
```
Comment (ipuustin) [inline crates/containerd-shim-wasmedge/src/executor.rs:48]: Maybe use `?` instead of `unwrap?`?

**Review by ipuustin — COMMENTED** [review]
I think this looks good, only one small nit.

Removing the async feature means that it isn't now present in either Wasmedge or Wasmtime shim. In theory we could enable one of them using the current wasmtime-fiber approach, or if wasmtime-fiber is going to be forked/replaced by Wasmedge soon enough we can just wait.

File: crates/containerd-shim-wasmedge/src/executor.rs (hunk 1 above)
Comment (CaptainVincent) [reply crates/containerd-shim-wasmedge/src/executor.rs]: Thanks your feedback.  Done.

**Comment by cpuguy83** [comment]
Thanks!

**Pull request merged**